src/CacheManager.Logging.SeriLog/project.json

```diff
@@ -0,0 +1,54 @@
+{
+  "version": "0.9.0-*",
+  "description": "CacheManager extension package providing SeriLog Logging for distributed caches. CacheManager is an open source caching abstraction layer for .NET written in C#. It supports various cache providers and implements many advanced features, the SeriLog logger can be used in place of the Microsoft Logging Extension",
+  "packOptions": {
+    "licenseUrl": "https://github.com/MichaCo/CacheManager/blob/master/LICENSE",
+    "projectUrl": "http://www.cachemanager.net",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/MichaCo/CacheManager"
+    },
+    "summary": "CacheManager extension package providing ProtoBuf serialization for distributed caches.",
+    "tags": [ "Caching", "Cache", "CacheManager", "Distributed Cache", "StackExchange Redis", "Azure AppFabric", "Memcached" ]
+  },
+  "authors": [ "Wenisman" ],
+  "buildOptions": {
+    "allowUnsafe": false,
+    "languageVersion": "csharp6",
+    "warningsAsErrors": true
+  },
+  "copyright": "Copyright (c) 2015 MichaConrad",
+  "configurations": {
+    "Debug": {
+      "buildOptions": {
+        "define": [ "DEBUG" ]
+      }
+    },
+    "Release": {
+      "buildOptions": {
+        "define": [ "RELEASE" ],
+        "optimize": true
+      }
+    }
+  },
+
+  "dependencies": {
+    "CacheManager.Core": "0.9.0-*",
+    "NETStandard.Library": "1.6.0",
```

Comment:
the netstandard library should be a dependency for netstandard framework only right?
If you put it here, it will be references by net45 and net452, too.
Btw, I'm not sure why you have both, the net45 and net452 trargets?